resources/migrations/000_migrations.yaml

```diff
@@ -8698,34 +8698,27 @@ databaseChangeLog:
                     nullable: true
 
 
-  - changeSet:
```

Comment:
I know you're removing this migration since it was a no-op, but you should probably comment on your own PR and call out things like this so people who didn't know that know why you're doing something like this know why.

When I open PRs I try to call out a bunch of stuff to help people reviewing them. Here's a recent example https://github.com/metabase/metabase/pull/27025

Reply:
Yeah, apologies for not adding a proper set of guiding comments. The only changes to migrations were there when I took over the branch so I can't really comment on this particular change, although I presume it's a no-op, like you said, that caused an error with H2 v2.